packages/timeline-state-resolver-types/src/superfly-timeline.ts

```diff
@@ -27,25 +27,23 @@ export interface ResolveOptions {
 	/** If set to true, the resolver will go through the instances of the objects and fix collisions, so that the instances more closely resembles the end state. */
 	resolveInstanceCollisions?: boolean
 }
-export interface TimelineObject {
+export interface TimelineObject<TContent = unknown> {
 	id: ObjectId
 	enable: TimelineEnable | TimelineEnable[]
 	layer: string | number
 	/** Group children */
-	children?: Array<TimelineObject>
+	children?: Array<TimelineObject<any>>
```

Comment:
I have a feeling I had an issue with `unknown` causing something to not work. Changing it now doesnt have any issue (unless the error was from blueprints/core), so lets start with `unknown`

Reply:
I found it. A load of the tests in `superfly-timeline.spec.ts` start failing if I change that.

I think that to fix this will require some additional larger changes to types, potentially including making supertimeline itself generic. As it's not an quick fix, I don't think I should spend more time on this.